src/main.ts

```diff
@@ -114,7 +114,7 @@ async function run(): Promise<void> {
  */
 
 function extractStoryIds(content: string): string[] {
-  const regex = /(?<=ch)\d{1,7}/g
+  const regex = /(?<=sc)\d{1,7}/g
```

Comment:
possiblement les 2 ?
```suggestion
  const regex = /(?<=(ch|sc))\d{1,7}/g
```

Reply:
J'ai changé